ui/components/requirements/requirement.jsx

```diff
@@ -7,6 +7,7 @@ export default function Requirement({ requirement }) {
     <span key={idx} className="req-text-line mb1">{ line }<br /></span>
   ));
   /* eslint-enable react/no-array-index-key */
+  const sunsetString = requirement.policy.sunset ? 'Sunset date by $requirement.policy.sunset' : 'Sunset date: none';
```

Comment:
@tadhg-ohiggins the way CM pointed out is a fancy new ECMAScript 6 way of doing it. They're called template literals https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Template_literals.